src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+    private static String color;
+
+    public static Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        color = colorSupplier.getRandomColor();
+        final int number = random.nextInt(100);

```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/097e649c-9eca-483e-87a5-96736fede234)

Reply:
Should i change final int number to private static final int number?

Reply:
number must be constant.
Read please, the article to understand what is constant and what is not
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names